I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 52, in script call
    call STORY
  File "game/STORY.rpy", line 283, in script call
    call gun_start
  File "game/gun-minigame.rpy", line 60, in script
    $ targetSM = SpriteManager(update=UpdateTarget)
  File "game/gun-minigame.rpy", line 60, in <module>
    $ targetSM = SpriteManager(update=UpdateTarget)
NameError: name 'UpdateTarget' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 52, in script call
    call STORY
  File "game/STORY.rpy", line 283, in script call
    call gun_start
  File "game/gun-minigame.rpy", line 60, in script
    $ targetSM = SpriteManager(update=UpdateTarget)
  File "renpy/ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/gun-minigame.rpy", line 60, in <module>
    $ targetSM = SpriteManager(update=UpdateTarget)
NameError: name 'UpdateTarget' is not defined

Windows-10-10.0.19041 AMD64
Ren'Py 7.6.1.23060707
DDLC - No More Time 0.0.1
Sat Jul  1 16:30:14 2023
